README.md

```diff
@@ -94,6 +80,8 @@ one or more of the following lines:
 ASSUMEROLE="IAM-role-arn"                      # IAM Role ARN for multi account. See below for more info
 IAM_AUTHORIZED_GROUPS="GROUPNAMES"             # Comma seperated list of IAM groups to import
 SUDOERS_GROUPS="GROUPNAMES"                    # Comma seperated list of IAM groups that should have sudo access
+IAM_AUTHORIZED_GROUPS_TAG="KeyTag"             # Key Tag of EC2 that contains a Comma seperated list of IAM groups to import
+SUDOERS_GROUPS_TAG="KeyTag"                    # Key Tag of EC2 that contains a Comma seperated list of IAM groups that should have sudo access
```

Comment:
if I understand this correctly, that SUDOERS_GROUPS_TAG  will override SUDOERS_GROUPS, so I can only use one of the config values. right?